mubench.reviewsite/src/RoutesHelper.php

```diff
@@ -29,6 +31,7 @@ public function __construct(DBConnection $db, PhpRenderer $renderer, Logger $log
         $this->site_base_url = $site_base_url;
         $this->upload_path = $upload_path;
         $this->default_ex2_review_size = $default_ex2_review_size;
+        $this->csv_helper = new CSVHelper();
```

Comment:
Please move all the download related stuff to a `DownloadController`, to avoid stuffing everything into this one bloated class.